### PR TITLE
Redirects and 404s

### PR DESCRIPTION
**Pull request by nottrobin**
QA
--

``` bash
./run
```

Check #291, #293 and #294 are fixed.

### CODE REVIEW

**Comment by nottrobin** [comment]
Oh and that Ubuntu bug: https://github.com/canonical-websites/www.ubuntu.com/issues/1873 - I redirected that URL too

File: templates/error/404.html [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-
-    <body>
-        <h1>404 - Page not found</h1>
-        <p>There is no page at this URL.</p>
-    </body>
-</html>
+{% block content %}
+<div class="col-12  p-layout__main">
+    <main class="p-layout__inner">
+        <h1 style="displa">404: Page not found</h1>
```
Comment (WillMoggridge) [inline templates/error/404.html]: Odd style tag included here

**Review by WillMoggridge — CHANGES_REQUESTED** [review]
Look good but one change request.

**Comment by WillMoggridge** [comment]
I worked out why the nav isn't showing, I'll PR the change separately

**Comment by WillMoggridge** [comment]
Created the PR #296

**Pull request merged**